templates/repo/issue/comment_tab.tmpl

```diff
@@ -5,11 +5,9 @@
 
 <div class="field">
 	{{template "shared/combomarkdowneditor" (dict
-		"MarkdownPreviewUrl" (print .Repository.Link "/markup")
-		"MarkdownPreviewContext" .RepoLink
-		"TextareaName" "content"
-		"TextareaContent" $textareaContent
-		"TextareaPlaceholder"  (ctx.Locale.Tr "repo.diff.comment.placeholder")
+		"CustomInit" true
+		"MarkdownPreviewInRepo" $.Repository "MarkdownPreviewMode" "comment"
+		"TextareaName" "content" "TextareaContent" $textareaContent "TextareaPlaceholder"  (ctx.Locale.Tr "repo.diff.comment.placeholder")
```

Comment:
split onto lines

Reply:
I have no motivation to do that, that's just personal preference.

If you strongly prefer to, feel free to edit this PR directly.